cmd/ocdb-put/main.go

```diff
@@ -120,15 +120,18 @@ func main() {
 	imax := 0
 	client := &http.Client{Timeout: 2 * time.Second}
 
-	_ = filepath.Walk(srcdir, func(path string, info os.FileInfo, err error) error {
+	err := filepath.Walk(srcdir, func(path string, info os.FileInfo, err error) error {
 		if strings.HasPrefix(filepath.Base(path), "Run") &&
 			filepath.Ext(path) == ".root" {
 			imax--
 			if imax == 0 {
-				return fmt.Errorf("toto")
+				return io.EOF
```

Comment:
@aphecetche : not completely sure about this one (and its handling below.)
could you check?

Reply:
Looks fine. 
I'm adding an option to control the number of processed files